pkg/forwarder/transaction/transaction.go

```diff
@@ -384,3 +384,17 @@ func (t *HTTPTransaction) SerializeTo(serializer TransactionsSerializer) error {
 	log.Trace("The transaction is not stored on disk because `storableOnDisk` is false.")
 	return nil
 }
+
+// truncateBodyForLog truncates body to prevent from logging a huge message
+// if body is more than 1000 bytes. 1000 bytes are enough to tell which tools a response comes.
+func truncateBodyForLog(body []byte) []byte {
+	if len(body) == 0 {
+		return []byte{}
+	}
+	limit := 1000
+	if len(body) < limit {
+		return body
+	}
+	log.Warn("truncated http response body to the first 1000 bytes to prevent from logging a huge message")
```

Comment:
I don't think this is the best solution. Some of the calls into this function are inside log.Tracef. We will be printing this warning even if the body is not actually logged. It might be better to append `...` or `(truncated)` or something similar to the body itself. WDYT?

Reply:
Appended `...` to the body itself. https://github.com/DataDog/datadog-agent/pull/10100/commits/c97ff7020ee319a9ca7c5279fffd8abf1f67ab0b

Reply:
I've found the example in another place.
Should I append `[]byte("...TRUNCATED...")` to the body itself to specify that the body is truncated?

https://github.com/DataDog/datadog-agent/blob/b5bed4dc6a5b08a50346e614b386ea67e37695de/pkg/logs/decoder/line_handler.go#L8-L10

https://github.com/DataDog/datadog-agent/blob/b5bed4dc6a5b08a50346e614b386ea67e37695de/pkg/logs/decoder/single_line_handler.go#L63-L78

Reply:
@vickenty 
I appreciate your review. Adding `...` is enough?

Reply:
Yes, I think this should be enough.